homeassistant/components/logi_circle/strings.json

```
@@ -11,19 +11,16 @@
         "description": "Please follow the link below and **Accept** access to your Logi Circle account, then come back and press **Submit** below.\n\n[Link]({authorization_url})"
       }
     },
-    "create_entry": {
-      "default": "Successfully authenticated with Logi Circle."
-    },
     "error": {
-      "auth_error": "API authorization failed.",
-      "auth_timeout": "Authorization timed out when requesting access token.",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "oauth2_authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
       "follow_link": "Please follow the link and authenticate before pressing Submit."
     },
     "abort": {
-      "already_setup": "You can only configure a single Logi Circle account.",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
       "external_error": "Exception occurred from another flow.",
       "external_setup": "Logi Circle successfully configured from another flow.",
-      "no_flows": "You need to configure Logi Circle before being able to authenticate with it. [Please read the instructions](https://www.home-assistant.io/components/logi_circle/)."
+      "oauth2_missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]"
```

We should use the key used in our helper, so if we use the helper in the future, it just works.
```suggestion
      "missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]"
```